support-frontend/assets/helpers/tracking/trackingOphan.ts

```diff
@@ -31,24 +30,44 @@ const trackBFCacheLoad = (): void =>
 		action: 'VIEW',
 	});
 
-export const buildOphanPayload = (participations: Participations): AbTest[] => {
+export const buildOphanPayload = (
+	participations: Participations,
+): AbTestRegister => {
 	const activeTests: Array<[string, string]> =
 		Object.entries(participations).filter(testIsActive);
 
-	return activeTests.map((participation) => ({
-		name: participation[0],
-		variant: participation[1],
-		complete: false,
-		campaignCodes: [],
-	}));
+	return activeTests.reduce<AbTestRegister>(
+		(
+			payload: AbTestRegister,
+			participation: [string, string],
+		): AbTestRegister => {
+			const ophanABEvent = {
+				variantName: participation[1],
+				complete: false,
+				campaignCodes: [],
+			};
+			return Object.assign({}, payload, {
+				[participation[0]]: ophanABEvent,
+			});
```

Comment:
Definitely subjective but would using a spread here be slightly simpler to read (while giving the same result)?

```
return {
  ...payload,
  [participation[0]]: ophanABEvent,
}
```

Reply:
yep have amended